src/Sylius/Bundle/CoreBundle/Resources/translations/messages.fr.yml

```diff
@@ -1,8 +1,19 @@
 # This file is part of the Sylius package.
-# (c) Sylius Sp. z o.o.
+# (c) Paweł Jędrzejewski
 
 sylius:
     email:
+        admin_password_reset:
+            hello: 'Bonjour'
```

Comment:
Wrapping strings in `' '` shouldn't be necessary, seems to be done only in this file.